aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,67 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
+[Use cases and example code](#when-to-use)  
```

Comment:
Would all of the "sign-up", "sign-in" etc be considered use cases?

Reply:
Didn't quite understand this. Are they not real-world examples? Is that what use cases are?